universe/interfaces.go

```diff
@@ -131,6 +132,8 @@ type World interface {
 
 	GetSettings() *WorldSettings
 
+	GetTotalStake() uint8
```

Comment:
I guess it should return `*big.Int`
Later we may refactor and migrate all tokens and stakes to uint256 go package https://pkg.go.dev/github.com/holiman/uint256